Fix for Wok RIS file parsing

jiaola opened the pull request:
When I tried to parse a ris file downloaded from Web of Science using RISparser 0.4.3, it fails for several reasons: 

1. It doesn't recognize the PT start tag. It throws a "KEY exception". 
2. It couldn't find the "is_counter" function in the Wok class
3. It fails to find the "ET" and "ER" tags if they don't have a trailing space. 

This pull request would solve the issues mentioned above.

shapiromatron commented:
Thanks for the PR! Could you add a few unit tests to show that it's now working?

jiaola commented:
I added an example and a test. I also added the mapping for WoS tags according to this: https://images.webofknowledge.com/images/help/WOS/hs_wos_fieldtags.html

shapiromatron commented:
Looks good, thanks!
